index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+    </section>
+
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+
+    <section class="container about_container">
+        <h1 class="text1">About Me</h1>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.
+        </p>
+        <a class="see_project3" href="#">Get My Resume</a>
+    </section>
+
+    <section class="about_container1">
+        <br>
+        <br>
+        <hr>
+        <br>
+        <br>
+        <div class="language_container">
+            <h2>
+                Language
+            </h2>
+            <br>
+            <p>Javascript</p>
+            <br>
+            <p>Ruby</p>
+            <br>
+            <p>Html</p>
+            <br>
+            <p>Css</p>
+        </div>
+        <br>
+        <br>
+        <br>
+        <div class="frameworks_container">
+            <h2>
+                Frameworks
+            </h2>
+            <br>
+            <p>Bootstrap</p>
+            <br>
+            <p>Ruby on Rails</p>
+            <br>
+            <p>RSPec</p>
+            <br>
+            <p>CapyBara</p>
+            <br>
+            <p>Selenium</p>
+        </div>
+        <br>
+        <br>
+        <br>
+        <div class="skills_container">
+            <h2>
+                Skills
+            </h2>
+            <br>
+            <p>Codekit</p>
+            <br>
+            <p>Github</p>
+            <br>
+            <p>Codepen</p>
+            <br>
+            <p>Gitlab</p>
+            <br>
+            <p>Terminal</p>
+        </div>
+    </section>
```

Comment:
- [ ] Kindly use css to add margin or padding this will make your project more semantic, It would be better to remove all `<br>`  from your project 👍🏿  

- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul` tag and  the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are in a `li` tag a list of the  `ul`, please remove the h2 and  p tags and use ul and li tags, this is part of the project requirements


 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![baa](https://user-images.githubusercontent.com/69638013/159151346-11d23d47-2719-4f2e-ba32-fde8bcc317fb.png)  | ![hmmmmm](https://user-images.githubusercontent.com/69638013/159151341-2c16a30b-7d78-4149-892c-a74f85924cad.png)
- [ ] Please add the appropriate icons for the new `ul` tags so your design looks like the Figma design 👍🏿